build.gradle

```diff
@@ -128,15 +128,6 @@ allprojects {
     }
 
     dependencies {
-        // Lombok dependencies.
-        def lombok = "org.projectlombok:lombok:1.18.30"
-        compileOnly lombok
-        annotationProcessor lombok
-        testCompileOnly lombok
-        testAnnotationProcessor lombok
-        testFixturesCompileOnly lombok
-        testFixturesAnnotationProcessor lombok
-
```

Comment:
Curious, what does removing this and pushing it down to connectors achieve? Not against it, especially if it's because they're only used in a few connectors.

This and the junit test deps are, in my book, OK to declare as global dependencies, because their existence or any version change won't meaningfully impact the published connectors.

Reply:
> Curious, what does removing this and pushing it down to connectors achieve? Not against it, especially if it's because they're only used in a few connectors.
> 
> This and the junit test deps are, in my book, OK to declare as global dependencies, because their existence or any version change won't meaningfully impact the published connectors.
> 

@postamar It's mainly due to how the Lombok and Micronaut annotation processors interact -- the Lombok one must be declared first.  Pushing it down to where it is used, while annoying, guarantees that we can maintain the correct order of the declared dependencies.  This is really temporary, as we should ditch Lombok entirely if/when we convert everything to Kotlin.